src/net.cpp

```diff
@@ -2063,20 +2063,15 @@ class CNetCleanup
 instance_of_cnetcleanup;
 
 
-
-
-
-
-
-void RelayTransaction(const CTransaction& tx)
+void RelayTransaction(const CTransaction& tx, CAmount fee)
```

Comment:
I mostly care about having it in the mempool, so the code can be reused by other logic than just feefilter.

Thanks for addressing the nits.